labscommunity/protocol-land
# feat: Show PR conflict message & disable merge on conflict

## Summary

This PR adds improvements to PR read page:
- To show PR conflict message & disable merge on conflict.
- PR updates (base & compare repo new update) check and update is done at an interval of 30 secs.
- On clicking the merge button, PR check and update is done before merging.

![image](https://github.com/labscommunity/protocol-land/assets/11836100/e024b4c7-0fd9-4658-8eb2-5f88b6458318)

## How to test
1) Make sure there is conflicts on branches that are merging to see the conflict message & merge disabled.
2) When there is merge conflict message shown, make update to the repo fixing the conflict using remote helper and wait until the check and update is done to remove the conflict message (done at 30secs interval).
3) Now you can merge.

> Note: Test both same and different repo(parent & fork) branches merge.
